skimage/transform/_warps.py

```diff
@@ -66,8 +66,8 @@ def resize(image, output_shape, order=None, mode='reflect', cval=0, clip=True,
     anti_aliasing : bool, optional
         Whether to apply a Gaussian filter to smooth the image prior
         to down-scaling. It is crucial to filter when down-sampling
-        the image to avoid aliasing artifacts. If input image data
-        type is bool, no anti-aliasing is applied.
+        the image to avoid aliasing artifacts. Default is True if down-scaling
```

Comment:
@mkcor, `anti_aliasing` is in fact set to True internally by default when down sizing. It is one of the additions of this PR (please see line 111) :wink:

Reply:
@rfezzani oh, thanks! Reading the PR description helps as well... :wink: I still think it's not accurate to write "Default is True" because that would mean a user could use a bool input image, not specify anti_aliasing, and fall under `if input_type == bool and anti_aliasing:` (which is not the case). I would therefore suggest the following rewording for this part of the docstring: "It is set to True when down-sizing an image of which data type is not bool."